packages/connect/src/device/DeviceList.ts

```diff
@@ -62,13 +61,16 @@ export class DeviceList extends TypedEmitter<DeviceListEvents> {
 
     transportFirstEventPromise: Promise<void> | undefined;
 
-    constructor() {
+    constructor({
+        transports,
+        debug,
+        messages,
+    }: {
+        transports?: ConnectSettings['transports'];
```

Comment:
maybe it wold be better to accept all settings here, like `constructor({ settings, messages })` 

DataManager is still used down in the code to get some other settings (priority, pendingTransportEvent) and this way it will be easier to get rid of it completely

Reply:
yes, but in this case I would need to assign these settings to `this`. So this is something I wanted to discuss or maybe do in a followup.

Reply:
removed in [2d3a470](https://github.com/trezor/trezor-suite/pull/12425/commits/2d3a47014e1412620b398d8518fbbf0ca234eea9)